Samples.Objc/Samples.ObjC/ViewControllers/AddItemViewController/Views/TextFieldCell.m

```diff
@@ -79,17 +83,4 @@ - (void)setupConstraints {
   [[self.textField.trailingAnchor constraintEqualToAnchor:self.contentView.trailingAnchor constant:-16] setActive:true];
   [[self.textField.bottomAnchor constraintEqualToAnchor:self.contentView.bottomAnchor constant:-16] setActive:true];
 }
-
-// MARK: - UITextFieldDelegate
-- (void)textFieldDidEndEditing:(UITextField *)textField {
-  [self.delegate cell:self textFieldDidEndEditing:self.textField.text];
-}
-
-- (BOOL)textFieldShouldReturn:(UITextField *)textField {
-  if (textField == self.textField) {
-    [textField resignFirstResponder];
-    return false;
-  }
-  return true;
```

Comment:
Will removing this cause any issues?

Reply:
ah my bad, this will make the default keyboard not disappearing when tapping done! 
I'll revert this function :smile: